core/deployment/src/main/java/io/quarkus/deployment/QuarkusAugmentor.java

```diff
@@ -141,6 +145,9 @@ public BuildResult run() throws Exception {
                 } catch (Exception e) {
                 }
             }
+            if (deploymentClassLoader instanceof Closeable) {
+                ((Closeable) deploymentClassLoader).close();
+            }
```

Comment:
Just wondering whether `classLoader` which is the application's augment classloader should also be closed here.

Reply:
No, they have different lifecycles. This one is recreated on each hot reload, the other lasts for the life of dev mode apps.